.changes/v3.1.0/602-improvements.md

```diff
@@ -0,0 +1,2 @@
+* `resource/vcd_catalog` allows to set and update `storage_profile_id`, supports update for `description` [GH-602]
```

Comment:
Let's split `description` update feature into a separate line - it deserves it! :)

Reply:
Done.